eo-maven-plugin/src/main/java/org/eolang/maven/OptimizeMojo.java

```diff
@@ -88,13 +93,12 @@ public final class OptimizeMojo extends SafeMojo {
     public void exec() {
         final long start = System.currentTimeMillis();
         final Collection<ForeignTojo> tojos = this.scopedTojos().withXmir();
-        final Optimization optimization = this.optimization();
         final int total = new Threaded<>(
             new Filtered<>(
                 ForeignTojo::notOptimized,
                 tojos
             ),
-            tojo -> this.optimized(tojo, optimization)
+            tojo -> this.optimized(tojo, this.optimization())
```

Comment:
@ArtemGet here, instead creating just one object you will create new object on every cycle iteration. So I believe this change is unnecessary

Reply:
Aggreed, reverted changes.